### DISCUSSION TITLE
Tutorial: Obtaining a Session Key for OpenAI Platform

### DISCUSSION THREAD

**Discussion by H0llyW00dzZ** [ChatGPTNextWeb/NextChat#2936] [Ideas]
In this tutorial, I will guide you on how to obtain a session key for the OpenAI platform. The session key can be used for any models (fine-tuning,playground,etc) if you already have an API key. However, we recommend using this session key specifically for the client app as a local.

The steps below to obtain the session key:

1. Go to the OpenAI platform by visiting the following URL: [https://platform.openai.com/](https://platform.openai.com/).
2. Once the page loads, you will need to inspect the element on the page. This can typically be done by right-clicking on the page and selecting "Inspect" or using the keyboard shortcut (e.g., Ctrl+Shift+I).
3. In the developer tools panel that appears, navigate to the "Network" tab.
4. Look for a request or response that contains the session key. It might be listed as a header or a parameter.
5. Once you locate the session key, you can copy it for further use.

Here's an example screenshot showing the session key in the developer tools panel:

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/1e608fd8-4a5c-4dc6-a697-e13d3cd8833a)

After obtaining the session key, you can use it in your client app as a local setting. This session key allows you to access and utilize OpenAI models for various purposes.

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/85abf275-2071-426e-9880-cf1eb5fed607)

> **Note:**
> The session key should be handled securely and not shared publicly. It provides access to your OpenAI resources and should be treated with care.

**Comment by llity** [comment]
我怎么显示额度是5美元呢？谢谢

**Reply by H0llyW00dzZ** [reply]
https://platform.openai.com/account/billing/limits

![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/17626300/c54e41b4-ceb2-4145-a8f6-bea3b34e90af)

**Reply by llity** [reply]
谢谢，只能付费账户用吗？想用免费的web版呢。

**Reply by H0llyW00dzZ** [reply]
about that if your openai api account already charged, you can set hard limit and soft limit.
